2 feat aloha 로봇팔 추가

BlindedShooter opened the pull request:
## 설명

<!--
  여러개의 Issue를 하나의 PR에 묶고 싶은 경우:
  "Issue Number: close " 옆에 Issue 번호를 함께 추가하세요. 자동으로 issue를 연결해줍니다.
  ex) Issue Number: close #23
-->

ALOHA 로봇팔과 그리퍼를 추가했습니다.

## 체크리스트

제출한 Pull Request가 다음 사항을 만족하는지 확인해주세요:

- [x] 👮 변경 내용이 규칙(lint rules, business rules 등)을 준수합니다.
- [x] 📝 bug fix나 feature 추가의 경우 변경 내용을 문서화하였습니다.
- [x] 😊 커밋 메시지가 변경 내용을 충분히 설명하고 있습니다.
- [x] 💯 코드를 추가/변경한 경우 기존 테스트를 잘 통과하고, 필요한 경우 적절한 테스트를 작성 하였습니다.
- [x] ❗ Breaking Change가 있는 경우, 외부로 노출된 API가 변경되지 않았고, 변경되었다면 migration guide가 있습니다.
<!--
  필요한 경우 체크리스트를 더 추가하세요. 본인 스스로 변경내용에 대해 다시 검토해볼 수 있고, 검토자에게 주의깊게 봐야할 사항을 알려줄 수 있습니다.
-->

## 추가 정보

<!--
  기능에 대한 추가적인 문맥이나, 스크린샷은 여기 추가해주세요.
-->

BlindedShooter committed:
Add RB3 to Robosuite

BlindedShooter committed:
Fix xml default parsing issue
and use <motor> instead of <general> in compliance with Robosuite requirement.

BlindedShooter committed:
Change default values

BlindedShooter committed:
Fix RB3 some joints not rotating & renaming
Some joint links were overlapping and thus not rotates. Adjusted z-axis offset of the base-shoulder link and every joints are fixed.

BlindedShooter committed:
Add simple test code

BlindedShooter committed:
Add ALOHA mjcf naively

BlindedShooter committed:
Update mjcf to follow Robosuite convention and add boilerplate

BlindedShooter committed:
Fix ALOHA arm to work

BlindedShooter committed:
Clean unused ALOHA mesh files

BlindedShooter committed:
Add ALOHA gripper & use as default in ALOHA arm

BlindedShooter committed:
Add ALOHA test code

powerstrong commented on robosuite/models/assets/robots/rb3/robot.xml:
```diff
@@ -0,0 +1,104 @@
+  </default>
+
+  <worldbody>
+  <body name="base" pos="0 0 0">
```
xml indent는 직접 넣으시는건가요? 자동으로 레이아웃 맞춰주는 도구가 있으면 좋을 것 같네요!

BlindedShooter replied:
넵 직접 넣었습니다! robosuite에서 필요한 wrapper인데, 굳이 중간에서 인덴트 한번 더 하기 싫어서 규칙을 어겼습니다 ^^7

powerstrong reviewed (approved):
LGTM 👍 수고하셨습니다

BlindedShooter committed:
Renamed wrist_cam_left to wrist_cam
Because this gripper is attached on both side and the suffix was redundant and misleading.
